src/test/java/com/training/baseTest/FlipkartBaseTest.java

```diff
@@ -0,0 +1,34 @@
+package com.training.baseTest;
```

Comment:
package name should not be in camel case. It should be all in lower case
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html